.circleci/config.yml

```diff
@@ -2,12 +2,17 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/node:12.0-browsers
+      - image: circleci/python:jessie-node-browsers
     steps:
       - checkout
       - run:
           name: Build and Test
           command: |
-            npm install
-            npm run build
-            npm run coverage
+            mkdir test-reports
+            sudo pip install selenium
+            sudo pip install requests
+            python test/test_selenium_chrome.py
+            python test/test_selenium_edge.py
+            python test/test_selenium_firefox.py
+            python test/test_selenium_IE.py
+            python test/test_selenium_safari.py
```

Comment:
is it possible to specify the browser version? or even run on multiple different versions on each browser?

Reply:
yes, we can define the platform, browser and version https://crossbrowsertesting.com/apidocs/v3/selenium.html#!/default/get_selenium_browsers